.github/workflows/functional.yml

```diff
@@ -0,0 +1,76 @@
+name: Functional Tests
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+env:
+   TEST_IMAGE_NAME: fiware/orion-ci:rpm7
+
+jobs:
+  functional-1:
+    runs-on: ubuntu-18.04
+
+    services:
+      mongodb:
+        image: mongo:4.0
+        ports:
+          - 27017:27017
```

Comment:
Is mongo really needed i this .yml? As far as I understand, MongoDB runs *inside* the fiware/orion-ci:rpm7 image (https://github.com/telefonicaid/fiware-orion/blob/master/ci/rpm7/build-dep.sh#L38)

(Similar comments for others .yml that use fiware/orion-ci:rpm7)

Reply:
When the mongo inside the image is running, no. But in general, its not good practice to have multiple processes running inside one docker container and therefore I would prefer to use the service container. But I forgot to remove it from the rpm image and would leave it up to you on how you prefer it.

Reply:
What about to leave it commented to be enabled once it gets removed from the rpm image? I'll do a separate comment suggestion for that.

For this thread, NTC.

Reply:
Sorry for the confusion, I made another commit to remove mongo from the image. Due to running the tests via 'build -miqts' (i.e. no -d parameter to start the db). they relied on the containerized mongo already. I added back the mongo-service